spec/models/results_post_processor_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe ResultsPostProcessor do
+  describe '#total_pages' do
```

Comment:
These tests violate the general 'one expectation per test' principle we try to follow, but any other way of doing this is likely going to overcomplicate (running through an array of possible page count/result combos in one expectation) or bloat the file (setting total pages in `let`s and doing one liner `it`s for the expectations).  It's fine as is, but just highlighting other possible approaches.  I'd probably have a slight preference for something like this, but your call:

```suggestion
  describe '#total_pages' do
    subject(:total_pages) { described_class.new.total_pages(total_results) }

    let(:total_results) { 0 }

    it { is_expected.to eq(0) }

    context 'when there is 1 result' do
      let(:total_results) { 1 }

      it { is_expected.to eq(1) }
    end

    context 'when there are 10 results' do
      let(:total_results) { 10 }

      it { is_expected.to eq(1) }
    end
    
    etc. etc.
```

Reply:
I'll clean this up.